src/styles/ui-components.scss

```diff
@@ -8,34 +8,32 @@
   overflow: hidden;
 }
 
-
 /* Begin Patternfly Toolbar overrides */
 
-.toolbar-pf .form-group .btn,
-.toolbar-pf .form-group .btn-group { /* overrides _toolbar.scss to account for intermediate Angular markup */
-  margin-left: 5px;
-}
 
-.form-group.miq-toolbar-group { /* styling for "Custom Html Toolbar Menu" example (used on classic-ui topology screen) */
-  display: inline-block;
-  .miq-custom-html {
-    form {
-      display: inline-block;
-    }
-    .form-group {
-      margin-bottom: 0 !important;
-      border-right: 0;
-      &:last-child {
-        padding-right: 0px
+/* overrides _toolbar.scss to account for intermediate Angular markup and adjusted the margins to properly style wrapped form-groups */
+
+.miq-toolbar-actions {
+  margin-top: -10px;
+  .form-group:first-of-type {
```

Comment:
This catches any first form group is this correct, because looks like search is swallowing this as well and it does not have `padding-left: 20px;` set up.
![screenshot from 2017-09-26 15-48-19](https://user-images.githubusercontent.com/3439771/30864182-39eb2c0e-a2d3-11e7-9065-67fd41eff3c3.png)

Reply:
@karelhala Good catch. I've corrected the styling for the custom toolbar and made an annotation to explain the form-group changes in the standard toolbar.